backend/pokebattle/settings/base.py

```diff
@@ -83,6 +84,11 @@ def base_dir_join(*args):
     "social_django",
 ]
 
+# REST_FRAMEWORK = {
+#     'DEFAULT_PERMISSION_CLASSES': [
+#         'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
+#     ]
+# }
```

Comment:
Why this comment?

Reply:
I deleted the comment, I saw this was necessary to set default permission, is this right?